terraform/modules/prom-ec2/alerts-config/alerts/govuk-accounts-alerts.yml

```diff
@@ -2,7 +2,7 @@ groups:
 - name: GovukAccounts
   rules:
   - alert: GovukAccounts_AppRequestsExcess5xx
-    expr: sum without(exported_instance, status_range) (rate(requests{org="govuk-accounts", space="production", status_range="5xx"}[5m])) / sum without(exported_instance, status_range) (rate(requests{org="govuk-accounts", space="production"}[5m])) >= 0.25
+    expr: sum without(exported_instance, status_range) (rate(requests{org="govuk-accounts", space="production", status_range="5xx", app!="govuk-account-static-errors"}[5m])) / sum without(exported_instance, status_range) (rate(requests{org="govuk-accounts", space="production"}[5m])) >= 0.25
```

Comment:
Does the denominator of the fraction also need an exclusion for the static errors app?  Otherwise the percent won't be quite right.

Reply:
No, Prometheus matches labels when doing arithmetic on metrics.  See https://prometheus.io/docs/prometheus/latest/querying/operators/#vector-matching

Reply:
:exploding_head: